app/helpers/application_helper.rb

```diff
@@ -1,2 +1,8 @@
 module ApplicationHelper
+
+  def full_title page_title = ""
+   base_title = "Ruby on Rails Tutorial Sample App"
```

Comment:
- indent = 2 spaces
```suggestion
    base_title = "Ruby on Rails Tutorial Sample App"
```

- apply I18n to translate this text `"Ruby on Rails Tutorial Sample App"`
ex:
```suggestion
   base_title = t "rails_app_title"
```